src/gobprepare/cloner/oracle_to_postgres.py

```diff
@@ -158,6 +159,60 @@ def _copy_data(self) -> int:
 
         return rows_copied
 
+    def _list_to_chunks(self, lst: list, chunk_size: int):
+        """Divides a list in chunks of chunk_size. Returns a list of dictionaries with "min" and "max" as keys, where
+        "min" is inclusive, and "max" is exclusive.
+
+        :param lst:
+        :param chunk_size:
+        :return:
+        """
+        result = []
+        min_idx = 0
+
+        while min_idx < len(lst):
+            row = {}
```

Comment:
```suggestion
            max_idx = min_idx + chunk_size
            result.append({
                'min': lst[min_idx] if min_idx > 0 else None,
                'max': lst[max_idx] if max_idx < len(lst) else None
            }
            min_idx = max_idx
```
and then after the loop: